modules/cluster-and-ha-management/pages/how_to-replace-a-node-in-a-cluster.adoc

```diff
@@ -2,7 +2,12 @@
 :description: This page describes the procedure to replace a node in a non-ha cluster.
 
 //welcome and introduction
-This guide outlines the procedure for replacing a node in a non-High Availability (HA) cluster. If your system uses xref:ha-overview.adoc[High Availability], refer to the documentation on removing a failed node in xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[].
+This guide outlines the procedure for replacing a node in a cluster regardless of whether it is an High Availability(HA) cluster. If your system uses xref:ha-overview.adoc[High Availability], refer to the documentation on removing a failed node in xref:tigergraph-server:cluster-and-ha-management:remove-failed-node.adoc[Removal of Failed Nodes].
+
+== Prerequisites
+* TigerGraph is installed with HostName, refer to xref:bare-metal-install.adoc[HostName Installation]. Additionally, TigerGraph should be installed on a device that can be unmounted from the exiting machine and mounted to the new machine.
```

Comment:
existing

Reply:
I meant to express the idea that the node is 'leaving' the cluster hence I called it 'exiting', but looks like it's not straightforward enough so I changed to `node to be replaced` and `to-be-replaced node`.